prov/tcp/src/tcpx_eq.c

```diff
@@ -52,6 +52,23 @@ static ssize_t tcpx_eq_read(struct fid_eq *eq_fid, uint32_t *event,
 	return ofi_eq_read(eq_fid, event, buf, len, flags);
 }
 
+static int tcpx_eq_close(struct fid *fid)
+{
+	struct tcpx_eq *eq;
+	int ret;
+
+	ret = ofi_eq_cleanup(fid);
+	if (ret)
+		return ret;
```

Comment:
What failure can occur here, and is this something the caller can realistically respond to?

Reply:
Util code returns BUSY indicating that ref count is not zero. I guess it may be useful for some apps?